### DISCUSSION TITLE
New kernel updates but where to download?

### DISCUSSION THREAD

**Discussion by 4integration** [microsoft/WSL#7502] [General]
Hi,

I see new kernel updates are available ( https://github.com/microsoft/WSL2-Linux-Kernel/releases )
When I download the update installer from https://docs.microsoft.com/sv-se/windows/wsl/install-manual#step-4---download-the-linux-kernel-update-package and run it, there is no update installed.

Currently I have v5.10.16.

How can I get the latest linux kernel?

**Comment by Biswa96** [comment]
The latest one is 5.10.60.1, here https://github.com/microsoft/WSL2-Linux-Kernel/releases

**Reply by 4integration** [reply]
Thanks @Biswa96 but that only contains source, no binaries

**Reply by Biswa96** [reply]
Did you install this from your provided documentation https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi

You can see the Linux kernel version with `uname -r` command.

**Reply by 4integration** [reply]
Yes, I installed via above exe, even uninstalled and installed again but still 5.10.16.3
```
$ uname -r
5.10.16.3-microsoft-standard-WSL2
```
